WebService/v1/Models/StatusApiModel.cs

```diff
@@ -15,8 +15,11 @@ public sealed class StatusApiModel
         [JsonProperty(PropertyName = "Name", Order = 10)]
         public string Name => "DeviceSimulation";
 
-        [JsonProperty(PropertyName = "Status", Order = 20)]
-        public string Status { get; set; }
+        [JsonProperty(PropertyName = "IsHealthy", Order = 20)]
+        public bool IsHealthy = true;
+
+        [JsonProperty(PropertyName = "Message", Order = 25)]
+        public string Message = "Alive and well!";
```

Comment:
Should this be StatusModel instead of separate properties?

Reply:
I thought about that. But it will be easier when other services check this json it will be easier to de-serialize if these properties are not nested.

Reply:
fixed